.github/workflows/bootstrap.yml

```diff
@@ -20,69 +20,46 @@ jobs:
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
-      # Restore the previous yarn modules and Cypress binary archives.
-      # Any updated archives will be saved automatically after the entire
-      # workflow successfully finishes.
-      # We use exact restore key to avoid yarn module snowballing
-      # https://glebbahmutov.com/blog/do-not-let-npm-cache-snowball/
-      - name: Cache central yarn modules
-        uses: actions/cache@v2
-        with:
-          path: ~/.cache/yarn
-          key: ${{ runner.os }}-node-${{ github.ref }}-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-node-${{ github.ref }}-${{ hashFiles('**/yarn.lock') }}
-
-      # We use the exact restore key to avoid Cypress binary snowballing
-      # https://glebbahmutov.com/blog/do-not-let-cypress-cache-snowball/
-      - name: Cache Cypress binary
-        uses: actions/cache@v2
+      - name: Setup Node
+        uses: actions/setup-node@v3
         with:
-          path: ~/.cache/Cypress
-          key: cypress-${{ runner.os }}-cypress-${{ github.ref }}-${{ hashFiles('**/package.json') }}
-          restore-keys: |
-            cypress-${{ runner.os }}-cypress-${{ github.ref }}-${{ hashFiles('**/package.json') }}
+          node-version-file: '.nvmrc'
+          cache: 'yarn'
 
-      - name: Cache dependencies
-        uses: actions/cache@v2
-        with:
-          path: '**/node_modules'
-          key: ${{ runner.os }}-modules-${{ hashFiles('yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-modules-${{ hashFiles('yarn.lock') }}
-
-      # Cache local node_modules to pass to testing jobs
-      - name: Cache local node_modules
+      - name: Cache Node Modules
         uses: actions/cache@v2
         with:
           path: node_modules
           key: ${{ runner.os }}-node-modules-${{ github.ref }}-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
             ${{ runner.os }}-node-modules-${{ github.ref }}-
 
-      - name: Cache build folder
-        uses: actions/cache@v2
+      - name: Install Cypress & Build Project 
+        uses: cypress-io/github-action@v3
         with:
-          path: build
-          key: ${{ runner.os }}-build-${{ github.ref }}-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-build-${{ github.ref }}-
+          # Disable running of tests within install job
+          runTests: false
+          build: yarn build
 
-      - name: install dependencies and verify Cypress
+      - name: Verify Cypress
         env:
           # Make sure every Cypress install prints minimal information
           CI: 1
         run: |
-          yarn install --frozen-lockfile
           npx cypress cache path
           npx cypress cache list
           npx cypress verify
           npx cypress info
 
-      - name: build
-        run: yarn run build
+      - name: Save Build Folder
+        if: ${{ !env.ACT }}
```

Comment:
We might want to add a comment here about how this (I assume) lets us test the workflow locally with `act`.